R/check_meta_col_name_duplicate.R

```diff
@@ -0,0 +1,61 @@
+#' Check there are no duplicated column names
+#'
+#' Make sure that all column names are unique.
```

Comment:
```suggestion
#' Make sure that all column names within the `col_name` column of the metadata file are unique.
```

Just a quick clarification on what we mean by column names, as there's also the column names in the data file itself that are checked separately